src/API/Test/InferenceRequestTest.cs

```diff
@@ -98,16 +98,62 @@ public void IsValidate_ShallReturnAllErrors()
             Assert.False(request.IsValid(out string _));
         }
 
+        [Fact(DisplayName = "IsValidate shall return false with invalid uri")]
+        public void IsValidate_ShallReturnFalseWithInvalidUri()
+        {
+            var request = new InferenceRequest();
+            request.InputResources.Add(new RequestInputDataResource
+            {
+                Interface = InputInterfaceType.Algorithm,
+                ConnectionDetails = new InputConnectionDetails()
+            });
+            request.InputResources.Add(new RequestInputDataResource
+            {
+                Interface = InputInterfaceType.DicomWeb,
+                ConnectionDetails = new InputConnectionDetails
+                {
+                    Uri = "http://this.is.not.a.valid.uri\\",
+                    AuthId = "token",
+                    AuthType = ConnectionAuthType.Bearer
```

Comment:
recommend adding a `,` to every line -- even the last item -- to avoid unnecessary line changes in future reviews.